### PR TITLE
Metal Ingots

### PR DESCRIPTION
**Pull request by Drew4484**
#### Summary
Content "Metal Ingots"

#### Purpose of change
Consolidates chunk/lumps into larger ingots for easier inventory management.

#### Describe the solution
Adds craftable 5L and 10L ingots for various metals. These can be reduced back to chunks/lumps, or can be used in recipes that require larger amounts of metal.

#### Describe alternatives you've considered
None.

#### Testing
WIP.

#### Additional context
As with most of our metalworking, these have a certain amount of handwaving to make them manageable. Normally these would be formed into long metal rods or billets but rods are meters long and would be a pain to store and rolled sheets are beyond a survivor's capability to manufacture.

### CODE REVIEW

File: data/json/items/resources/metal.json [hunk 1]
```diff
@@ -589,5 +589,455 @@
     "color": "light_gray",
     "ammo_type": "components",
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "id": "5kgaluminum",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": { "str_sp": "5 l aluminum ingot" },
+    "description": "A 5 L ingot of solid aluminum.",
```
Comment (TheShadowFerret) [inline data/json/items/resources/metal.json:598]:
```suggestion
    "name": { "str_sp": "5L aluminum ingot" },
    "description": "A five-liter ingot of solid aluminum.",
```
Proper gramma'

File: data/json/items/resources/metal.json [hunk 2]
```diff
@@ -589,5 +589,455 @@
...
+    "symbol": "=",
+    "color": "light_gray",
+    "ammo_type": "components"
+  },
+  {
+    "id": "10kgaluminum",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": { "str_sp": "10 l aluminum ingot" },
+    "description": "A 10 L ingot of solid aluminum.",
```
Comment (TheShadowFerret) [inline data/json/items/resources/metal.json:613]:
```suggestion
    "name": { "str_sp": "10L aluminum ingot" },
    "description": "A ten-liter ingot of solid aluminum.",
```

**Review by TheShadowFerret — CHANGES_REQUESTED** [review]
Grammar check
*applies to all the metals, just made two examples

File: data/json/items/resources/metal.json [hunk 3]
```diff
@@ -589,5 +589,455 @@
     "color": "light_gray",
     "ammo_type": "components",
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "id": "5kgaluminum",
```
Comment (TheShadowFerret) [inline data/json/items/resources/metal.json:594]: Kg is weight, not volume 🤔

Comment (Drew4484) [reply data/json/items/resources/metal.json:594]: Oops, I'll fix this in a bit.

File: data/json/items/resources/metal.json (hunk 1 above)
Comment (Qrox) [reply data/json/items/resources/metal.json:598]:
```suggestion
    "name": { "str": "5L aluminum ingot" },
    "description": "A five-liter ingot of solid aluminum.",
```
Use "str" to generate the correct plural. (And the same for other items.)

**Comment by TheShadowFerret** [comment]
Is there anything left keeping on draft?

**Comment by Drew4484** [comment]
> Is there anything left keeping this on draft? Seems complete to me

I need to add recipes, got distracted trying to get another project ready for freeze but I'll wrap this one up ASAP.

**Pull request merged**